tests/algorithms/csm/test_walsh.py

```diff
@@ -1,20 +1,20 @@
 """Tests the iterative Walsh algorithm."""
 
 import torch
-from mrpro.algorithms.csm import iterative_walsh
+from mrpro.algorithms.csm import walsh
 from mrpro.data import SpatialDimension
 from tests.algorithms.csm.conftest import multi_coil_image
 from tests.helper import relative_image_difference
 
 
-def test_iterative_Walsh(ellipse_phantom, random_kheader):
-    """Test the iterative Walsh method."""
+def test_Walsh(ellipse_phantom, random_kheader):
```

Comment:
while you are at it: i would not capitalize walsh here, as the function is also named "walsh" not "Walsh"